src/tests/userIterraction.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+
+describe('Calculator', () => {
+  let getByTestId; let
+    getByText;
+
+  beforeEach(() => {
+    const component = render(<Calculator />);
+    getByTestId = component.getByTestId;
+    getByText = component.getByText;
+  });
+
```

Comment:
- [ ] Awesome job implementing the RTL, however, we also need to write test for the `modulus %` operator as we did for the other operators